src/framing/plaintext.rs

```diff
@@ -179,7 +185,7 @@ impl MlsPlaintext {
         credential: &Credential,
     ) -> Result<(), VerificationError> {
         let tbs_payload = MlsPlaintextTbs::new_from(&self, Some(serialized_context));
-        tbs_payload.verify(credential, &self.signature)
+        tbs_payload.verify(credential).map_err(|e| e.into())
```

Comment:
Nitpick: Maybe make this into `...?;` followed by `Ok()` just to be consistent in style? Not that we're really being consistent anywhere else. Feel free to ignore, I guess...